src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,14 @@
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-wrap: wrap;
+  max-width: 750px;
```

Comment:
How did U find this value? 
![Screenshot_4](https://user-images.githubusercontent.com/110488158/194068392-ade01ef4-964c-4723-a549-8fc1abbbb140.png)